test/slow/api_spec_slow.js

```diff
@@ -156,6 +157,20 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
     });
   });
 
+  describe('import', () => {
+    before(async () => {
+      await ensureTestDirIsNonexistent();
+      await fs.mkdir(dir);
+      execSync(`${installer} init -y`, {
+        cwd: dir,
+      });
+    });
+
+    it('works', async () => {
+      await forge.import({ dir });
```

Comment:
Seems like there should be some assertions in here.

Reply:
Do you have a suggestion for what to assert? I was kind modeling this test after [this](https://github.com/electron-userland/electron-forge/blob/master/test/slow/api_spec_slow.js#L121-L126) one. I could definitely see adapting [this](https://github.com/electron-userland/electron-forge/blob/master/test/slow/api_spec_slow.js#L159) suite to run after import as well as after init, but that's more work than I personally have time to put in now, and one simple sanity test is better than zero, no? But if you have a suggestion for one or several simple assertions to drop into this test, I'm happy to do so.

Reply:
Hmm yeah, those need better tests too :smile: (in another PR)

Let me think about what I want to do here and I'll make some changes in the branch if I have time this week, if you don't mind.

Reply:
I don't mind at all!